src/answers.sql

```diff
@@ -1,15 +1,89 @@
 -- Your answers here:
 -- 1
-
+SELECT C.name,
+       COUNT(*)
+FROM states AS S
+INNER JOIN countries AS C ON S.country_id = C.id
+GROUP BY C.id,
+         C.name;
+         
 -- 2
+SELECT COUNT(*)
+FROM employees
+WHERE supervisor_id IS NULL; 
 
 -- 3
+SELECT C.name,
+       O.address,
+       count(*) AS count
+  FROM offices AS O
+  INNER JOIN countries AS C ON O.country_id = C.id
+  LEFT JOIN employees AS E ON O.id = E.office_id
+GROUP BY C.name,
+         O.address
+ORDER BY 3 DESC,
+         C.name
+LIMIT 5;
 
 -- 4
+SELECT S.ID, CONCAT(S.first_name, ' ', S.last_name) AS full_name,
+       COUNT(*) AS count
+FROM employees AS E
+INNER JOIN employees AS S ON E.supervisor_id = S.id
+GROUP BY S.id, S.first_name, S.last_name
+ORDER BY 3 DESC;
 
 -- 5
+SELECT count(*) AS count
+FROM offices AS O
+INNER JOIN states AS S ON O.state_id = S.id
+INNER JOIN countries AS C ON O.country_id = C.id
+WHERE S.name = 'Colorado'
+  AND C.name = 'United States';
 
 -- 6
+SELECT O.name,
+       COUNT(e.id) AS COUNT
+FROM offices AS O
+LEFT JOIN employees AS E ON O.id = E.office_id
+GROUP BY O.ID,
+         O.name
+ORDER BY 2 DESC;
```

Comment:
You are right; I only used that notation because I found it interesting to use ORDER BY with ordinal notation to simplify or speed up writting some queries. However, in the long term, I believe it is better to use the column name as you suggested. I've changed it, in this commit: https://github.com/ravnhq/DB-Nerdery-Challenges/pull/50/commits/2c1da063b20a9325b1a5c56278947cd17733bc89